galaxy/tools/deps/mulled/mulled_build.py

```diff
@@ -144,6 +145,15 @@ def mull_targets(
         '-set', "BINDS='%s'" % bind_str,
         command,
     ]
+    if test_files:
+        test_bind = []
+        for test_file in test_files:
+            if ':' not in test_file:
```

Comment:
Should we also test if the file exists and raise a warning if this file does not exists?